radio/src/simu.cpp

```diff
@@ -445,7 +445,7 @@ void OpenTxSim::updateKeysAndSwitches(bool start)
   SWITCH_KEY(C, 2, 3);
   SWITCH_KEY(D, 3, 3);
 
-  #if defined(RADIO_TPRO) || defined(RADIO_TPROV2) || defined(RADIO_TPROS)
+  #if defined(RADIO_TPRO) || defined(RADIO_TPROV2) || defined(RADIO_TPROS) || defined(RADIO_BUMBLEBEE)
```

Comment:
This is for function switches, isn't it? i.e. TPRO S shouldn't be here?

Reply:
> This is for function/customisable switches, isn't it? i.e. TPRO S shouldn't be here?

I'm afraid it is not; it defines the 'regular) switch types for the simu

Reply:
Not quite. It actually does trigger some of the customisable switches, but only if those switches haven't been reached. i.e. For T-Pro 3 to 6 toggle the first 4 customisable switches. For TPro V2, 5 and 6 do the first two, etc. Just seems odd to switch to using 1-6 as the triggers for the (sometimes optional) E-F-G-H keys of those radios. But not relevant for this PR.